migrations/sql/V2019.04.25.11.30__add_variance_applications_metadata.sql

```diff
@@ -0,0 +1,105 @@
+CREATE TABLE variance_application_status_code (
+    variance_application_status_code character varying(3)                   NOT NULL,
+    description                      character varying(100)                 NOT NULL,
+    active_ind                       boolean                  DEFAULT true  NOT NULL,
+    create_user                      character varying(60)                  NOT NULL,
+    create_timestamp                 timestamp with time zone DEFAULT now() NOT NULL,
+    update_user                      character varying(60)                  NOT NULL,
+    update_timestamp                 timestamp with time zone DEFAULT now() NOT NULL,
+
+    UNIQUE(variance_application_status_code)
+);
+
+ALTER TABLE variance_application_status_code OWNER TO mds;
+
+COMMENT ON TABLE variance_application_status_code IS 'The valid options for a variance / variance application.';
+
+INSERT INTO variance_application_status_code (
+    variance_application_status_code,
+    description,
+    create_user,
+    update_user
+)
+VALUES
+    ('REV', 'In Review', 'system-mds', 'system-mds'),
+    ('NAP', 'Not Applicable', 'system-mds', 'system-mds'),
+    ('APP', 'Approved', 'system-mds', 'system-mds'),
+    ('DEN', 'Denied', 'system-mds', 'system-mds');
+
```

Comment:
I think this should be moved to seed data unless needed otherwise.

Reply:
Sure. What's the proposed benefit of having seed data in that single file?

Reply:
We have talked about wanting to reuse that seed data file for the test db, which may or may not be necessary given our testing overhaul @james-hollinger can chime in. 

Also means we can just change the existing seed_data file if we need to add an option to this. (removing an option would be tricky). also keeps all the type/code data in one place.

Reply:
The reason I ask is two-fold:
1. I seem to remember there was a time before launch when we did a full database teardown to resolve an issue with database seeds + versioning that wouldn't have happened if the seeds were done in in versioned migration files. I could be misremembering this.

2. I'm not convinced of the benefit of maintaining a large and growing seed data file. For example, the combination of commodities and compliance codes would have made for a substantial seed data file on their own.

Reply:
While not all of our static data is in one place, it really should be. We shouldn't have to trawl through 30 files just to see what the default data in our db's code tables are.

As well, if its all in one place we could theoretically drop and recreate the test db easier for testing in the future. The test db shouldn't be defined by constants files, it should be getting the same static data our app is getting.

Reply:
1. I don't recall seed data being the issue, but i'm not confident that it wasn't either. 

One large files that contains the seed data for the tables, separate from the table definitions themselves seems like it would be preferred over having data and structure distributed through dozens or hundreds of sql files.

Reply:
Okay. I'll concede